util/findLostPatches.py

```diff
@@ -0,0 +1,178 @@
+from jira.client import JIRA
```

Comment:
the python scripts naming convention is lower case, so this should be findlostpatches.py